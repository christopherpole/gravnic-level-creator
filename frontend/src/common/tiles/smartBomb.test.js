import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import SmartBombTile from './smartBomb';

configure({ adapter: new Adapter() });

describe('<SmartBombTile />', () => {
  it('Renders without exploding', () => {
    const smartBombTile = shallow(<SmartBombTile />);

    expect(smartBombTile).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const smartBombTile = shallow(<SmartBombTile />);

    expect(toJson(smartBombTile)).toMatchSnapshot();
  });
});
