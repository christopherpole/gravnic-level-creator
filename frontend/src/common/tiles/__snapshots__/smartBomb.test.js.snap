// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SmartBombTile /> Matches the current snapshot 1`] = `<styled.div />`;
